components/board/pi/impl/board.go

```diff
@@ -202,6 +202,18 @@ func (pi *piPigpio) Reconfigure(
 	return nil
 }
 
+// StreamTicks starts a stream of digital interrupt ticks.
+func (pi *piPigpio) StreamTicks(ctx context.Context, interrupts []string, ch chan board.Tick, extra map[string]interface{}) error {
+	for _, name := range interrupts {
+		interrupt, ok := pi.DigitalInterruptByName(name)
+		if !ok {
+			return errors.Errorf("unknown digital interrupt: %s", name)
```

Comment:
Same suggestion here re: trouble if the second interrupt returns errors but the first already has the callback added.